Server/mods/deathmatch/logic/CPedSync.cpp

```diff
@@ -86,25 +90,20 @@ void CPedSync::UpdatePed(CPed* pPed)
     // This ped got a syncer?
     if (pSyncer)
     {
-        // He isn't close enough to the ped and in the right dimension?
-        if ((!IsPointNearPoint3D(pSyncer->GetPosition(), pPed->GetPosition(), (float)g_TickRateSettings.iPedSyncerDistance)) ||
-            (pPed->GetDimension() != pSyncer->GetDimension()))
-        {
-            // Stop him from syncing it
-            StopSync(pPed);
+        // Is he close enough, and in the right dimension?
+        if (IsPointNearPoint3D(pSyncer->GetPosition(), pPed->GetPosition(), (float)g_TickRateSettings.iPedSyncerDistance))
+            if (pPed->GetDimension() == pSyncer->GetDimension())
+                return;
```

Comment:
1. I think blocks without curly braces should be generally avoided.
2. I think instead of nested if it's better to use &&.

```
if (IsPointNearPoint3D(pSyncer->GetPosition(), pPed->GetPosition(),(float)g_TickRateSettings.iPedSyncerDistance))
&& (pPed->GetDimension() == pSyncer->GetDimension())
{
    return;
}
```

Reply:
Blocks without curly braces are fine, as long as they are not nested.

Please refer to our [style guide](https://github.com/multitheftauto/mtasa-blue/wiki/Style-Guide) for more information.